docs/design/versioning.md

```diff
@@ -34,6 +34,8 @@ Documentation for other releases can be found at
 
 # Kubernetes API and Release Versioning
 
+Reference: http://semver.org
```

Comment:
Agreed.  I generally do as well.  Occasionally it's useful to expose a full but short URL.
No strong opinion either way here.